addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -731,21 +731,21 @@
                 <a t-attf-href="/jobs?all_countries=1#{non_country_params}"
                     t-attf-class="dropdown-item d-flex align-items-center justify-content-between nav-link#{'' if country_id or is_remote else ' active'}">
                     All Countries
-                    <span t-attf-class="badge rounded-pill#{' bg-primary' if country_id or is_remote else ' bg-light text-primary'} ms-2" t-out="count_per_country.get('all', '0')"/>
+                    <span t-attf-class="badge rounded-pill text-bg-primary ms-2" t-out="count_per_country.get('all', '0')"/>
```

Comment:
No need to differentiate `country_id or is_remote` or not anymore?

Reply:
See [this comment.](https://github.com/odoo/odoo/pull/162935#discussion_r1706970291)